.github/workflows/smoke_test.yml

```diff
@@ -24,41 +24,36 @@ jobs:
       - name: Setup Run Smoke Tests
         uses: ./.github/actions/setup_run_smoke_tests
 
-      - name: Run succeeding test
-        env:
-          TRUNK_PUBLIC_API_ADDRESS: https://api.trunk-staging.io
-        shell: bash
-        run: |
-          ./trunk-analytics-cli test \
-          --org-url-slug trunk-staging-org \
-          --junit-paths ${{ github.workspace }}/target/**/*junit.xml \
-          --token ${{ secrets.TRUNK_STAGING_ORG_API_TOKEN }} \
-          cargo nextest run -p smoke-test --profile ci
-
-  slack-workflow-status:
-    if: always() && needs.build_cli.result != 'success'
-    name: Post Smoke Test Failure
-    needs:
-      - build_cli
-    runs-on: ubuntu-latest
-    permissions:
-      actions: read
-    env:
-      SLACK_BOT_TOKEN: ${{ secrets.TRUNKBOT_SLACK_BOT_TOKEN }}
-    steps:
-      - name: Analytics Cli Smoke Test Failure
-        uses: slackapi/slack-github-action@v1
+      - name: Run Smoke Tests
+        uses: ./.github/actions/perform_smoke_test
         with:
-          channel-id: production-notifications
-          payload: |
-            {
-              "blocks": [
-                {
-                  "type": "section",
-                  "text": {
-                    "type": "mrkdwn",
-                    "text": "Release ${{ env.RELEASE }} of the analytics cli is broken! The job attempting a simple upload had a result of ${{ needs.build_cli.result }}. Runs can be found at https://github.com/trunk-io/analytics-cli/actions/workflows/smoke_test.yml"
-                  }
-                }
-              ]
-            }
+          cli-binary-location: trunk-analytics-cli
+          staging-api-token: ${{ secrets.TRUNK_STAGING_ORG_API_TOKEN }}
+          production-api-token: ${{ secrets.TRUNK_PROD_ORG_API_TOKEN }}
+#  slack-workflow-status:
```

Comment:
Commented out so I can test the prod runs without blowing up notifications channels.

Reply:
When do you plan to uncomment?